specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/stable/2020-05-15/timeseriesinsights.json

```diff
@@ -0,0 +1,2549 @@
+        ],
+        "operationId": "Environments_Update",
+        "x-ms-examples": {
+          "EnvironmentsUpdate": {
+            "$ref": "./examples/EnvironmentsPatchTags.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "description": "Updates the environment with the specified name in the specified subscription and resource group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/EnvironmentNameParameter"
+          },
+          {
+            "name": "environmentUpdateParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/EnvironmentUpdateParameters"
+            },
+            "description": "Request object that contains the updated information for the environment."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The environment definition was successfully updated and is in the response. If the environment was updated synchronously, the response will include a provisioningState value of \"Succeeded\". If the environment was updated asynchronous, the response will include a provisioningState value of \"Updating\".  You can periodically get your environment definition and monitor progress of the update via the provisioningState property.",
+            "schema": {
+              "$ref": "#/definitions/EnvironmentResource"
+            }
+          },
+          "default": {
+            "description": "HTTP 404 (Not Found): The subscription, resource group, or environment could not be found.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Environments"
+        ],
+        "operationId": "Environments_Delete",
+        "x-ms-examples": {
+          "EnvironmentsDelete": {
+            "$ref": "./examples/EnvironmentsDelete.json"
+          }
+        },
+        "description": "Deletes the environment with the specified name in the specified subscription and resource group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/EnvironmentNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The environment was successfully deleted."
+          },
+          "204": {
+            "description": "The environment was successfully deleted."
```

Comment:
>The environment was successfully deleted. [](start = 28, length = 41)

204 is supposed to be returned when the resource doesn't exist at the time of the request. Can you update the description?